### PR TITLE
[WFLY-17871] added test-case for managed executor service defined via…

### PR DESCRIPTION
**Pull request by istudens**
… annotation

This is a test-case for managed executor service defined via annotations. I can't see any such test in the test-suite.

Issue: https://issues.redhat.com/browse/WFLY-17871

### CODE REVIEW

**Comment by kabir** [comment]
@istudens I see a failure in the new test case https://ci.wildfly.org/buildConfiguration/WF_PullRequest_GalleonLinuxJdk11/366184?expandBuildProblemsSection=true&hideProblemsFromDependencies=false&expandBuildTestsSection=true&hideTestsFromDependencies=false&expandBuildChangesSection=true

**Comment by darranl** [comment]
@emmartins it looks like this one is waiting for a review from you?

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationTestCase.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationTestCase.java:21]: Please update to the current copyright/license header.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java:21]: Please update to the current copyright/license header.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import jakarta.ejb.Stateless;
+import jakarta.enterprise.concurrent.ContextServiceDefinition;
+import jakarta.enterprise.concurrent.ManagedExecutorDefinition;
+import jakarta.enterprise.concurrent.ManagedExecutorService;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import static jakarta.enterprise.concurrent.ContextServiceDefinition.APPLICATION;
+import static jakarta.enterprise.concurrent.ContextServiceDefinition.SECURITY;
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java:36]: import static should come first, then import java, then the rest.

**Review by bstansberry — CHANGES_REQUESTED** [review]
@istudens Formatting stuff aside, this is fine. If you see this and update the formatting stuff, please ping me in zulip so I know to check on it.

**Comment by istudens** [comment]
@bstansberry done

**Review by bstansberry — APPROVED** [review]

**Review by emmartins — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @istudens

**Pull request merged**